### DISCUSSION TITLE
DOIs at least for releases from tools with publications

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-tools#256] [General]
Hi Folks,
I came across this [Zenodo](https://zenodo.org/) and this [how to guide](https://guides.github.com/activities/citable-code/) for how to connect it to your GitHub account and organizations. It doesn't help us with the desire/promise [to mint DOIs for riverscapes projects in the warehouse](https://riverscapes.xyz/Data_Warehouses/#advantages). However, it does help with developers wanting to mint DOIs for specific releases. In my opinion, this is a practice we should encourage anytime we publish a manuscript vetting a tool, that a DOI is also minted for that release.  I think it is unnecessary and overkill for every release. 

I'd be curious:
1. @MattReimer or @philipbaileynar how hard does this look to you? I made account and it was easy... 
2. Is this reasonable?
3. Downsides
3.  Other thoughts?

**Comment by philipbaileynar** [comment]
The how to guide you referenced looks like a great way to mint a DOI for an official software tool release cited in a publication.
